lib/group-pull-requests.js

```diff
@@ -0,0 +1,54 @@
+const compareVersions = require("compare-versions");
+
+const groupPullRequests = (pullRequests) => {
+
+    const dependabotPRs = {}
+    const otherPRs = []
+
+    const bumpVersionRegExp = new RegExp("Bump ([^\S]+) from ([^\S]+) to ([^\S]+)", "g");
+
+    // group all matching dependabot PRs
+    for (let i in pullRequests) {
+        const currentPR = pullRequests[i]
+
+        if (currentPR.title.match(bumpVersionRegExp)) {
```

Comment:
maybe faster to check simply check PR author is dependabot first and then check if title match.